torchelastic/rendezvous/etcd_rendezvous.py

```diff
@@ -1119,13 +1121,23 @@ def _etcd_rendezvous_handler(url):
         etcd_prefix - path prefix (from etcd root), inside which all
                       etcd nodes will be created.
                       Default is "/torchelastic/p2p".
+        protocol=https - http (default) or https to access etcd.
+        cacert=/etc/kubernetes/certs/ca.crt - CA cert to access etcd,
+                    only makes sense with https.
+        cert=/etc/kubernetes/certs/client.crt - client cert to access etcd,
+                    only makes sense with https.
+        key=/etc/kubernetes/certs/client.key - client key to access etcd,
+                    only makes sense with https.
+
     """
     import re
     from urllib.parse import urlparse
 
     url = urlparse(url)
     assert url.scheme == "etcd"
 
+    kwargs = {}
```

Comment:
can we move the declaration + initialization of `kwargs = {}` down to line 1169 so that it is closer where it is used? and also extract the `kwargs` parsing to a method (e.g. `parse_etcd_client_params(params) -> Dict)`

Reply:
done

Reply:
awesome looks great! thanks! will merge to trunk.